src/GenericFactory.sol

```diff
@@ -147,6 +155,9 @@ contract GenericFactory is IGenericFactory, Owned {
         // TODO: Test that _loadCurve errors for invalid indexes.
         bytes memory lInitCode = _loadCurve(aCurveId, lToken0, lToken1);
 
+        // SAFETY:
+        // This assembly block is memory safe as no previous memory is overwritten and
+        // the free memory pointer is still correct
```

Comment:
Statement is weird, it should just be "this assembly block does not write to memory"

Reply:
ah okay. 

I put the phrasing as the following to accentuate the fact that this block IS memory safe. 
```
// This assembly block is memory safe as it does not write to memory
```